config/api.yml

```diff
@@ -3233,6 +3233,8 @@
       :post:
       - :name: cancel
         :identifier: miq_request_control
+      - :name: execute
+        :identifier: miq_request_control
```

Comment:
Continung from https://github.com/ManageIQ/manageiq-api/pull/1249#discussion_r1500946506

I still don't like this permission here...something feels off about it.  I admit that miq_request_control is used for cancel, but being able to cancel your own request is very different from executing it.

Reply:
added 'you can only execute this if it has been approved' feels like it lessens the request.